README.md

```diff
@@ -2,9 +2,9 @@
 
 [![Four Kitchens](https://img.shields.io/badge/4K-Four%20Kitchens-35AA4E.svg)](https://fourkitchens.com/)
 
-<h4 align="center">Customizable style guide + Storybook Development + Drupal 8 theme</h4>
+<h4 align="center">Storybook component library + Webpack build system + Drupal 8 theme</h4>
 
-Emulsify is a full Design System encompassing both a [Storybook](https://storybook.js.org/)/[Webpack](https://webpack.js.org/) development environment, a customizable Styleguide powered by [GatsbyJS](https://www.gatsbyjs.org/) (coming soon), and a Drupal 8 starterkit theme. It can be used as a standalone prototyping tool or inside a Drupal installation. It supports both Twig and React components!!
+Emulsify Drupal is a part of the [Emulsify Design System](https://github.com/emulsify-ds) and provides a [Storybook](https://storybook.js.org/) component library, a [Webpack](https://webpack.js.org/) development environment and a Drupal 8 starterkit theme. It can be used as a standalone prototyping tool or inside a Drupal installation. It supports both Twig and React components!!
```

Comment:
Add this to the end:

> It supports both Twig and React components out of the box. Many other languages are supported through plugins.

Reply:
I think this is actually complicated when speaking about `Emulsify Drupal` instead of the design system as a whole. `Emulsify Drupal` currently has a dependency on Storybook React and is specifically geared to Twig and/or React.

Reply:
That makes sense. I'm fine with leaving it as-is.